app/src/main/res/layout/fragment_full_announcement.xml

```diff
@@ -91,12 +91,20 @@
                 android:paddingTop="10dp"
                 >
 
-                <TextView
-                    android:id="@+id/announcement_content"
+                <ScrollView
```

Comment:
What is the `ScrollView` needed for inside the card?

Reply:
it makes scrolling in the announcement body much smoother (ie. u can flick up and down to scroll)

Reply:
Oh shoot I didn't notice this is the full card, my bad, this isn't an issue lol.